docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: "3.5"
+
+services:
+  rabbitmq:
+    image: rabbitmq:3.8.2-alpine
+    ports:
+      - "5672:5672"
+      - "15672:15672"
+    volumes:
+      - ./.docker/rabbitmq/etc/:/etc/rabbitmq/
+      - ./.docker/rabbitmq/data/:/var/lib/rabbitmq/
```

Comment:
a tutaj użyłbym wolumenu zamiast zmontowania folderu.

https://docs.docker.com/engine/reference/commandline/volume_create/

te foldery są spoko jak przekazujesz sobie na bieżąco jakieś dane kontener => system plików na kompie.
w momencie jak tworzysz sobie jakąś przestrzeń do przechowywania danych raczej używa się wolumenów.

tutaj jest nawet na stronie dockera jeszcze kilka argumentów:
https://docs.docker.com/storage/volumes/

"Volumes are the preferred mechanism for persisting data generated by and used by Docker containers" - pierwsze linijki tekstu